src/RF24Logger.h

```diff
@@ -22,6 +22,13 @@
 #include "RF24LogLevel.h"
 #include "RF24LogHandler.h"
 
+#if defined (ARDUINO_ARCH_AVR)
+  #define RF24LOG_FLASHIFY(A) F(A)
+  #define RF24LOGGER_info(vendorId, message, ...) (rf24Logger.log(RF24LogLevel::INFO, RF24LOG_FLASHIFY(vendorId), RF24LOG_FLASHIFY(message), ##__VA_ARGS__))
+#elif
+  #define RF24LOGGER_info(vendorId, message, ...) (rf24Logger.log(RF24LogLevel::INFO, vendorId, message, ##__VA_ARGS__))
+#endif

```

Comment:
On non AVR architectures the macro will pass the arguments just are they are, so as a char pointers (I hope - this is not tested yet).